libvirt.go

```diff
@@ -521,6 +521,11 @@ func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
 	}
 	var s []Lease
 
+	// In case of status file is empty then don't try to unmarshal data
+	if len(data) == 0 {
```

Comment:
What happens if `data` is `nil`? Will go return 0, or will it crash?

Reply:
@cfergeau it will never crash since data is type `[]byte` 

- https://play.golang.org/p/aDhOe8nAr4r

Reply:
Good, thanks :)